spec/view.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow } from 'enzyme';
+import { shallowToJson } from 'enzyme-to-json';
+import Photo from '../client/src/components/Photo.jsx';
+
+const photoUrl = 'https://cap-bnb-photo-service.s3.us-east-2.amazonaws.com/1-stream-belmont-3.jpg';
+
+configure({ adapter: new Adapter() });
+
+describe('Photos', () => {
```

Comment:
Okay, this is a good start! But after the above revisions, you're going to have more than just a Photo component rendering, and we should test all of the components that we are adding to the DOM.

Now that you know how to do a snapshot test, can you write one for each component?

Bonus points if you can write a test that mounts an outer component, and then checks to see if it renders the appropriate children!

Reply:
Heard loud and clear